x/ccv/provider/keeper/relay.go

```diff
@@ -270,18 +238,10 @@ func (k Keeper) QueueVSCPackets(ctx sdk.Context) {
 	k.IncrementValidatorSetUpdateId(ctx)
 }
 
-// EndBlockCIS contains the EndBlock logic needed for
-// the Consumer Initiated Slashing sub-protocol
-func (k Keeper) EndBlockCIS(ctx sdk.Context) {
-	// set the ValsetUpdateBlockHeight
-	blockHeight := uint64(ctx.BlockHeight()) + 1
-	valUpdateID := k.GetValidatorSetUpdateId(ctx)
-	k.SetValsetUpdateBlockHeight(ctx, valUpdateID, blockHeight)
-	k.Logger(ctx).Debug("vscID was mapped to block height", "vscID", valUpdateID, "height", blockHeight)
-
-	// Replenish slash meter if necessary, BEFORE executing slash packet throttling logic.
-	// This ensures the meter value is replenished, and not greater than the allowance (max value)
-	// for the block, before the throttling logic is executed.
+// BeginBlockCIS contains the BeginBlock logic needed for the Consumer Initiated Slashing sub-protocol.
+func (k Keeper) BeginBlockCIS(ctx sdk.Context) {
```

Comment:
Slash meter replenishment now happens on beginblock, since we want to replenish the meter before slash packets are handled DURING the block. Whereas previously all slash packets were handled in endblocker after the replenishment call